AssetEditor/AssetEditor.csproj

```diff
@@ -46,6 +46,7 @@
 -->
     <GenerateDocumentationFile>False</GenerateDocumentationFile>
     <NoWarn>$(NoWarn),1573,1591,1712</NoWarn>
+    <BaseOutputPath>C:\Users\Josh E\Downloads</BaseOutputPath>
```

Comment:
Avoid checking in changes which are only related to your personal computer as it will cause problems for others.